src/containers/EditSubjectFrontpage/components/nodes/NodeSearchDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useTranslation } from 'react-i18next';
+
+import { Node } from '@ndla/types-taxonomy';
+
+import SearchDropdown from './SearchDropdown';
+import { useSearchNodes } from '../../../../modules/nodes/nodeQueries';
+import { useTaxonomyVersion } from '../../../StructureVersion/TaxonomyVersionProvider';
+
+interface Props {
+  onChange: (t: any) => void;
+  selectedItems?: Node[];
+  wide?: boolean;
+}
+
+const NodeSearchDropdown = ({ onChange, selectedItems, wide }: Props) => {
+  const { t } = useTranslation();
+
+  const { taxonomyVersion } = useTaxonomyVersion();
```

Comment:
Denne gjør ingenting, da vi ikke har en `TaxonomyVersionProvider` lenger opp i treet. Kan like så godt bruke default-verdien isåfall